apps/sensenet/src/context/CurrentAncestors.tsx

```diff
@@ -78,7 +80,10 @@ export const CurrentAncestorsProvider: React.FunctionComponent = props => {
   }, [currentContent.Id, loadLock, reloadToken, repo])
 
   if (error) {
-    throw error
+    logger.warning({
+      message: `Error loading ancestors. ${error.toString()}`,
```

Comment:
Imo it would be better to show only the error message here. all other things are passed in the details.
```suggestion
      message: `Error loading ancestors. ${error.message}`,
```